commercial/conf/routes

```diff
@@ -3,53 +3,53 @@
 # ~~~~
 
 # For dev machines
-GET         /assets/*path                                                 dev.DevAssetsController.at(path)
-GET         /commercial/test-page                                         controllers.commercial.CreativeTestPage.allComponents(k: List[String])
-GET         /_healthcheck                                                 controllers.HealthCheck.healthCheck()
+GET         /assets/*path                                                       dev.DevAssetsController.at(path)
+GET         /commercial/test-page                                               controllers.commercial.CreativeTestPage.allComponents(k: List[String])
+GET         /_healthcheck                                                       controllers.HealthCheck.healthCheck()
 
 # Travel offer merchandising components
-GET         /commercial/travel/offers.json                                controllers.commercial.TravelOffersController.renderTravel
-GET         /commercial/travel/api/offers.json                            controllers.commercial.TravelOffersController.getTravel
+GET         /commercial/travel/offers.json                                      controllers.commercial.TravelOffersController.renderTravel
+GET         /commercial/travel/api/offers.json                                  controllers.commercial.TravelOffersController.getTravel
 
 # Job merchandising components
-GET         /commercial/jobs.json                                         controllers.commercial.JobsController.renderJobs
-GET         /commercial/jobs/api/jobs.json                                controllers.commercial.JobsController.getJobs
+GET         /commercial/jobs.json                                               controllers.commercial.JobsController.renderJobs
 
 # Masterclass merchandising components
-GET         /commercial/masterclasses.json                                controllers.commercial.MasterclassesController.renderMasterclasses
+GET         /commercial/masterclasses.json                                      controllers.commercial.MasterclassesController.renderMasterclasses
 
 # Soulmates merchandising components
-GET        /commercial/soulmates/$subgroup<\w+>.json                      controllers.commercial.SoulmatesController.renderSoulmates(subgroup)
-GET        /commercial/soulmates/$subgroup<\w+>                           controllers.commercial.SoulmatesController.getSoulmates(subgroup)
+GET        /commercial/soulmates/$subgroup<\w+>.json                            controllers.commercial.SoulmatesController.renderSoulmates(subgroup)
+GET        /commercial/soulmates/api/$subgroup<\w+>.json                        controllers.commercial.SoulmatesController.getSoulmates(subgroup)
```

Comment:
is this a merge problem? This line was changed earlier: https://github.com/guardian/frontend/pull/14457